### PR TITLE
add stub Bible v2

### PR DESCRIPTION
**Pull request by yadutaf**
We now have Bible section. This update was one of the most praised in
the application history! We want to take it to next level and allow for
early feedback without breaking the main features.

This commit introduces a "clean-sheet" new section to experiment with a
new way to integrate the Bible in the application.

This new section is hidden behind a 'secret' tester option to limit the
impact on existing users.

### CODE REVIEW

**Comment by nath5394** [comment]
I skimmed the modifications, this is a significant job you've done ! I can't understand the purpose of every bit of this code. Nevertheless it looks clean, it works on your tablet and on my phone, so I merge the PR.

**Comment by yadutaf** [comment]
I Like your proposition for the psalms numbering, inluding adding the "heb." prefix. Could you give me the full correspondance between AELF's psalm names and what should be displayed ? I'm not an expert of this and I'd rather avoid making a mistake.

No emergency on this area though, there is still a *LOT* of work !

**Comment by yadutaf** [comment]
**Changes**:
* Fixed crash on restore bible state
* Add "ripple effect" on click (nothing more :))

**Known Issue**:
* The active table (At/NT/Psaumes) is not restored.

**Comment by nath5394** [comment]
For the psalms, you can find here infos for numeration : https://fr.wikipedia.org/wiki/Livre_des_Psaumes#D%C3%A9coupage_et_num%C3%A9rotation is enough for you ? 
I'll test your last commit soon.

File: app/src/main/java/co/epitre/aelf_lectures/bible/BibleBookList.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+        BiblePart biblePart = new BiblePart("Psaumes");
+        BibleSection bibleSection = new BibleSection("Psaumes");
+        for (int i = 1; i<= 150; i++) {
+            if (i == 9 || i == 113) {
+                bibleSection.addBook(new BibleBook("Psaume "+i+"A"));
+                bibleSection.addBook(new BibleBook("Psaume "+i+"B"));
+            } else {
+                bibleSection.addBook(new BibleBook("Psaume "+i));
+            }
```
[the remark this answers is not in the document]
Comment (yadutaf) [reply app/src/main/java/co/epitre/aelf_lectures/bible/BibleBookList.java]: This is a stub. I'm wiring the GUI for the moment.

**Comment by yadutaf** [comment]
The code is starting to look pretty good from my point of view. I'd like to merge it to get some feedback from early testers.

Here is the current state of things:
* The refactored Bible replaces the current implementation, but only when a specific preference is enabled (off by default)
* The theme is a cleaned version of the office theme so that it looks clean. In a future PR, the office code should be shared with this one.
* Share support is implemented
* Opening a link is implemented

**Pull request merged**